src/main/java/seedu/address/logic/parser/profile/ViewProfilesCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser.profile;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_OPTION;
+
+import seedu.address.logic.commands.profile.ViewProfilesCommand;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ViewProfilesCommand object
+ */
+public class ViewProfilesCommandParser implements Parser<ViewProfilesCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ViewProfilesCommand
+     * and returns a ViewProfilesCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ViewProfilesCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        String viewProfilesOption = PREFIX_OPTION + ViewProfilesCommand.COMMAND_OPTION;
```

Comment:
Perhaps this could be declared as a `static final` variable since the value will not change for each execution?